### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by Shiuki**
Some fixes

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ This article describes some of the limits, specifications, and other requirement
 |Number of members in an [org-wide team](create-an-org-wide-team.md) | 10,000       |
 |Number of teams a global admin can create        |  500,000   |
 |Number of teams a Microsoft 365 or Office 365 organization can have    | 500,000&sup3;     |
-|Number of channels per team    | 200 (includes deleted channels)&sup4;         |
-|Number of Private channels per team    |30| (includes deleted channels)&sup4;
+|Number of channels per team    | 200 (includes deleted channels)<sup>4</sup>;         |
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:40]:
```suggestion
|Number of channels per team    | 200 (includes deleted channels)<sup>4</sup>        |
```

File: Teams/limits-specifications-teams.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ This article describes some of the limits, specifications, and other requirement
...
+|Number of Private channels per team    |30 (includes deleted channels)<sup>4</sup>;
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:41]:
```suggestion
|Number of Private channels per team    |30 (includes deleted channels)<sup>4</sup>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please commit my minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**